configuration.rst

```diff
@@ -409,17 +409,18 @@ The files stored in ``config/packages/`` are used by Symfony to configure the
 the application behavior by changing which configuration files are loaded.
 That's the idea of Symfony's **configuration environments**.
 
-A typical Symfony application begins with three environments: ``dev`` (for local
-development), ``prod`` (for production servers) and ``test`` (for
-:doc:`automated tests </testing>`). When running the application, Symfony loads
+A typical Symfony application begins with three environments:
+* ``dev`` for local development,
+* ``prod`` for production servers,
+* ``test`` for :doc:`automated tests </testing>`.
+When running the application, Symfony loads
 the configuration files in this order (the last files can override the values
 set in the previous ones):
 
-#. ``config/packages/*.yaml`` (and ``*.xml`` and ``*.php`` files too);
-#. ``config/packages/<environment-name>/*.yaml`` (and ``*.xml`` and ``*.php`` files too);
-#. ``config/services.yaml`` (and ``services.xml`` and ``services.php`` files too);
-#. ``config/services_<environment-name>.yaml`` (and ``services_<environment-name>.xml``
-   and ``services_<environment-name>.php`` files too).
+#. The files in ``config/packages/``;
+#. the files in ``config/packages/<environment-name>/``;
+#. ``config/services.yaml`` (and ``services.xml`` and ``services.php``);
+#. ``config/services_<environment-name>.yaml`` (and its XML and PHP version).
```

Comment:
I think we should keep supported extension in `config/packages/*`

Maybe something like this
```suggestion
#. The files in ``config/packages/*.<extension>``;
#. the files in ``config/packages/<environment-name>/*.<extension>``;
#. ``config/services.<extension>``;
#. ``config/services_<environment-name>.<extension>``.
```

And add a sentence like "Supported extensions are ``php``, ``yaml`` and ``xml``"

Reply:
I deleted it on purpose, cause this 3-formats-thing is repeated over and over on that page - while other things are missing completely ;-)
But yours is certainly better than the original!